### PR TITLE
WELD-2016 More meaningful package version information in manifest

### PR DESCRIPTION
**Pull request by mkouba**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -385,13 +391,10 @@
                <configuration>
                   <archive>
                      <manifestEntries>
-                        <Implementation-URL>${project.url}</Implementation-URL>
```
Comment (jharting) [inline pom.xml]: Do we want to remove URL and vendor from the manifest entirely?

Comment (mkouba) [inline pom.xml]: `Implementation-Vendor` and `Specification-Vendor` are currently empty strings. The added value of having this defined is questionable. `Implementation-URL` is deprecated and should define "the URL from which the extension implementation can be downloaded from". See also https://docs.oracle.com/javase/8/docs/technotes/guides/jar/jar.html#JAR_Manifest. I find the meaning of these attributes rather unclear and therefore I'd give as little information as possible.

Comment (jharting) [inline pom.xml]: Agreed. Thanks for explanation.

**Comment by jharting** [comment]
Merged, thanks!

**Pull request closed without merging**